experiments/testing_archive/test_tx_beam_order_bad_index.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
```

Comment:
This file is missing a test entry in the csv file

Reply:
This tests the same thing as `test_tx_beam_order_listnum_too_high`, so I will remove it.